Makefile

```diff
@@ -32,34 +32,35 @@ define LDFLAGS
 "
 endef
 
+BUILDTAGS = containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
```

Comment:
Can we call this something like "SKOPEO_BUILDTAGS"?

Reply:
I think `BUILDTAGS` is ok.  They do have global effect.

Reply:
Right, but we might have buildtags of our own eventually
Adding SKOPEO there would make it clear where they came from